importer/test_data/struct_expression.asm

```diff
@@ -7,9 +17,16 @@ mod other {
         a: int,
         b: E1,
     }
+
+    struct S2 {
+        x: Option<int>,
+        y: S1,
+    }
 }
 
 use other::E1;
 use other::S1;
+use other::S2;
 
-let s1 = S1{ a: 1, b: E1::A };
\ No newline at end of file
+let s1 = S1{ a: 1, b: E1::A };
```

Comment:
Please also add a struct to the prelude. If you don't then the change above to `check_path_try_prelude` is not tested.